main.tf

```diff
@@ -174,6 +174,8 @@ resource "azurerm_linux_function_app" "mcd_agent_service" {
   resource_group_name = azurerm_resource_group.mcd_agent_rg.name
   location            = azurerm_resource_group.mcd_agent_rg.location
 
+  builtin_logging_enabled = false
```

Comment:
nit: might be worth adding a comment explaining why this is needed. In case a future dev is adding a new function. The name of this variable is not obvious that it can kill the whole function randomly 😅

Reply:
good point @sassmith , will add a comment